packages/web3torrent/src/library/web3torrent-lib.ts

```diff
@@ -256,14 +256,19 @@ export default class WebTorrentPaidStreamingClient extends WebTorrent {
   protected async transferFunds(wire: PaidStreamingWire) {
     const channelId = wire.paidStreamingExtension.peerChannelId;
 
-    await this.paymentChannelClient.makePayment(
-      channelId,
-      WEI_PER_BYTE.mul(BUFFER_REFILL_RATE).toString()
-    );
-    const newSeederBalance = bigNumberify(
-      this.paymentChannelClient.channelCache[channelId].beneficiaryBalance
-    );
-    log(`payment made for channel ${channelId}, newSeederBalance: ${newSeederBalance}`);
+    if (
+      this.paymentChannelClient.channelCache[channelId] &&
+      this.paymentChannelClient.channelCache[channelId].status === 'running'
```

Comment:
In what scenario would this method, `transferFunds`, be called where there is not a running channel?

Reply:
When the channel is `closed`, for example.